AllReadyApp/Web-App/AllReady/Features/Notifications/NotifyAdminForActivitySignup.cs

```diff
@@ -27,13 +27,16 @@ public void Handle(VolunteerInformationAdded notification)
             var volunteer = _context.Users.Single(u => u.Id == notification.UserId);
             var activity = _context.Activities.Single(a => a.Id == notification.ActivityId);
             var campaign = _context.Campaigns
-                .Include(c => c.Organizer)
+                .Include(c => c.CampaignContacts).ThenInclude(cc => cc.Contact)
                 .Single(c => c.Id == activity.CampaignId);
             var link = $"View activity: http://{_options.Value.SiteBaseUrl}/Admin/Activity/Details/{activity.Id}";
 
             var subject = $"A volunteer has signed up for {activity.Name}";
 
-            if (campaign.Organizer != null)
+            var campaignContact = campaign.CampaignContacts.SingleOrDefault(tc => tc.ContactType == (int)ContactTypes.Primary);

```

Comment:
Though it's not functionally possible right now with user workflows, handle the possibility that `CampaignContacts` could be null. `var campaignContact = campaign.CampaignContacts?.SingleOrDefault(...)`